ui/src/components/pages/Login/index.tsx

```diff
@@ -10,21 +10,20 @@ export function Login() {
   const returnURL = searchParams.get("returnUrl") || "/";
 
   const handleLoginClick = useCallback(async () => {
-    try {
-      const response = await fetch(`/api/v1/login?returnUrl=${returnURL}`, {
-        method: "GET",
-        headers: {
-          "Content-Type": "application/json",
-        },
-      });
-      const error = await getAPIResponseError(response);
-      if (error) {
-        setError(error);
-      }
-    } catch (e) {
-      setError(e);
-    }
-  }, []);
+    // try {
+    //   const response = await fetch(`/api/v1/login?returnUrl=${returnURL}`);
+    //   if (response.ok) {
+    window.location.replace(`/api/v1/login?returnUrl=${returnURL}`);
```

Comment:
Should the returnURL be determined by the dex server? The app (Numaflow api server) should have a config in place with the dex server so that the dex server knows which redirect url to use after a successful authentication.

Reply:
This PR is not final work. It is just some shared starting code for mocking. 

This returnUrl is the url we want the user to eventually come back to once they are logged in. Once dex is integrated there will be other redirects before we come back to returnUrl.